full/src/main/java/apoc/load/LoadDirectory.java

```diff
@@ -109,9 +112,11 @@
 
     // visible for test purpose
     public static String checkIfUrlBlankAndGetFileUrl(String urlDir) throws IOException {
-        return StringUtils.isBlank(urlDir)
-                ? encodePath(getDirImport())
-                : FileUtils.changeFileUrlIfImportDirectoryConstrained(urlDir.replace("?", "%3F"));
+        if (StringUtils.isBlank(urlDir)) {
+            final Path pathImport = Paths.get(getDirImport()).toAbsolutePath();
+            return chop(encodePath(pathImport.toUri().toString()));
+        }
```

Comment:
perché?

Reply:
Perché con windows come stava prima dava un `Failed to invoke procedure apoc.load.directory: Caused by: java.nio.file.FileSystemNotFoundException: Provider "C" not installed`.
Il chop serve solo per togliere lo `/` alla fine del path (es: `file:///C:/ILMIOPATH/`), ma in realtà anche senza funziona comunque, ma mi sembra più corretto.